.github/workflows/kindIntegTest.yml

```diff
@@ -231,23 +231,26 @@ jobs:
     strategy:
       matrix:
         version:
-          - "4.0.15"
-          - "4.1.7"
-          - "5.0.2"
-          - "6.8.53"
-          - "6.9.6"
+          - "4.0.17"
+          - "4.1.8"
+          - "5.0.3"
+          - "6.8.54"
+          - "6.9.7"
+          - "5.1.0"
         integration_test:
           - test_all_the_things
           - smoke_test_read_only_fs
         include:
-          - version: 6.8.53
-            serverImage: datastax/dse-mgmtapi-6_8:6.8.53-ubi8 # DSE 6.8.53
+          - version: 6.8.54
+            serverImage: datastax/dse-mgmtapi-6_8:6.8.54-ubi8 # DSE 6.8.54
             serverType: dse
-          - version: 6.9.6
-            serverImage: datastax/dse-mgmtapi-6_8:6.9.6-ubi # DSE 6.9.6
+          - version: 6.9.7
+            serverImage: datastax/dse-mgmtapi-6_8:6.9.7-ubi # DSE 6.9.7
             serverType: dse
+          - version: 5.1.0
```

Comment:
comment:
Hopefully trunk doesn't become 6.0 soon, but we'll deal with that if/when it happens

Reply:
It shouldn't affect the test itself even if the version changes as there would be no difference in processing 5.1.0 vs 6.0. The version just needs to be hardcoded to something here.